src/Raven.Client/Documents/Linq/RavenQueryProviderProcessor.cs

```diff
@@ -1360,6 +1360,13 @@ private void VisitMethodCall(MethodCallExpression expression)
                 }
                 return;
             }
+
+            if (declaringType == typeof(MemoryExtensions))
```

Comment:
Can we add a test which uses that explicitly?

Reply:
Done